bin/device_setup.sh

```diff
@@ -70,14 +50,6 @@ while sleep 1; do
 
     kick_rebooter $d
```

Comment:
unless you install the rebooter app on emulators, this can probably be removed, together with the function definition above.

Reply:
Done